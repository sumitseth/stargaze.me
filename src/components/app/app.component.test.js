import React from 'react';
import { render, getByTestId } from '@testing-library/react';
import renderer from 'react-test-renderer';
import AppComponent from './app.component';
import { appTestSelectors } from './test.selectors';

it('renders without crashing', () => {
  const { container } = render(<AppComponent />);
  const title = getByTestId(container, appTestSelectors.title);
  expect(title.textContent).toBe('stargaze.me');
});

it('matches snapshot', () => {
  const tree = renderer.create(<AppComponent />).toJSON();
  expect(tree).toMatchSnapshot();
});
