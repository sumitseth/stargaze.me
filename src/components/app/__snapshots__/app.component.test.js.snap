// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="App"
  data-testid="app-container"
>
  <header
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src="logo.svg"
    />
    <p
      data-testid="app-title"
    >
      stargaze.me
    </p>
  </header>
</div>
`;
